### PR TITLE
UBO fixes

### PR DESCRIPTION
**Pull request by alexcristici**

### CODE REVIEW

**Review by sjg-wdw — APPROVED** [review]

**Review by TimSylvester — APPROVED** [review]

File: include/mbgl/shaders/gl/drawable_fill_outline.hpp [hunk 1]
```diff
@@ -11,7 +11,6 @@ struct ShaderSource<BuiltIn::FillOutlineShader, gfx::Backend::Type::OpenGL> {
     static constexpr const char* vertex = R"(layout (std140) uniform FillOutlineDrawableUBO {
     highp mat4 u_matrix;
     highp vec2 u_world;
-    highp vec2 pad;
```
Comment (stefankarschti) [inline include/mbgl/shaders/gl/drawable_fill_outline.hpp:14]: This is a needed padding field to keep the ubo size multiple of 16 bytes.

Comment (alexcristici) [reply include/mbgl/shaders/gl/drawable_fill_outline.hpp:14]: Then I probably need to add it in the glsl file to in sync.

File: src/mbgl/renderer/layers/render_line_layer.cpp [hunk 2]
```diff
@@ -485,50 +485,92 @@ void RenderLineLayer::update(gfx::ShaderRegistry& shaders,
 
         // interpolation UBOs
         const float zoom = static_cast<float>(state.getZoom());
```
Comment (stefankarschti) [inline src/mbgl/renderer/layers/render_line_layer.cpp:487]: `zoom` could change every frame, therefore the interpolation UBOs could also be different for data driven properties. Not updating these after the first compute could be affecting visuals?

Comment (alexcristici) [reply src/mbgl/renderer/layers/render_line_layer.cpp:487]: This code is running per frame, so the zoom is updated per frame.

Comment (stefankarschti) [reply src/mbgl/renderer/layers/render_line_layer.cpp:487]: correct, my bad.

**Review by stefankarschti — APPROVED** [review]

**Pull request merged**